pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [
     {include = "**/*.py", from = "src"},
 ]
 readme = "README.md"
-version = "0.7.8"
+version = "0.8.0"
```

Comment:
Application code that relied on the old `PASS/FAIL` `result.label` will break! Technically, it should be a major version update, but we're not ready for 1.0.0 yet.